docs/getting-started/backends-and-brokers/sqs.rst

```diff
@@ -192,6 +192,27 @@ The above policy:
 +-----------------------------------------+--------------------------------------------+
 
 
+SQS - STS token authentication
+------------------------
```

Comment:
```suggestion
STS token authentication
----------------------------
```

Please ensure the title and the header notation are at the same length.
Otherwise, Sphinx produces warnings.
Also, in my opinion, adding SQS to the title is redundant.